docs/sources/microsoft-365/msft-teams/example-api-responses/sanitized/Communications_callRecords_v1.0.json

```diff
@@ -1,226 +1,54 @@
 {
-  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#communications/callRecords(sessions(segments()))/$entity",
-  "version":1,
-  "type":"peerToPeer",
-  "modalities":[
-    "audio"
-  ],
-  "lastModifiedDateTime":"2020-02-25T19:00:24.582757Z",
-  "startDateTime":"2020-02-25T18:52:21.2169889Z",
-  "endDateTime":"2020-02-25T18:52:46.7640013Z",
-  "id":"e523d2ed-2966-4b6b-925b-754a88034cc5",
-  "organizer":{
-    "user":{
-      "id":"821809f5-0000-0000-0000-3b5136c0e777",
-      "tenantId":"dc368399-474c-4d40-900c-6265431fd81f"
-    }
-  },
-  "participants":[
-    {
-      "user":{
-        "id":"821809f5-0000-0000-0000-3b5136c0e777",
-        "tenantId":"dc368399-474c-4d40-900c-6265431fd81f"
-      }
-    },
-    {
-      "user":{
-        "id":"f69e2c00-0000-0000-0000-185e5f5f5d8a",
-        "tenantId":"dc368399-474c-4d40-900c-6265431fd81f"
-      }
-    }
-  ],
-  "sessions":[
+  "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#communications/callRecords",
+  "value":[
     {
+      "id":"3cf3bbc8-b21d-4f2e-bfd0-b13603ae6c65",
+      "version":2,
+      "type":"unknown",
       "modalities":[
         "audio"
       ],
-      "startDateTime":"2020-02-25T18:52:21.2169889Z",
-      "endDateTime":"2020-02-25T18:52:46.7640013Z",
-      "id":"e523d2ed-2966-4b6b-925b-754a88034cc5",
-      "isTest":false,
-      "caller":{
-        "@odata.type":"#microsoft.graph.callRecords.participantEndpoint",
-        "cpuName":"Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz",
-        "cpuCoresCount":2,
-        "cpuProcessorSpeedInMhz":2594,
-        "userAgent":{
-          "@odata.type":"#microsoft.graph.callRecords.clientUserAgent",
-          "headerValue":"RTCC/7.0.0.0 UCWA/7.0.0.0 AndroidLync/6.25.0.27 (SM-G930U Android 8.0.0)",
-          "platform":"android",
-          "productFamily":"skypeForBusiness"
-        },
-        "identity":{
-          "@odata.type":"#microsoft.graph.identitySet",
-          "user":{
-            "id":"821809f5-0000-0000-0000-3b5136c0e777",
-            "tenantId":"dc368399-474c-4d40-900c-6265431fd81f"
-          }
+      "lastModifiedDateTime":"2023-09-25T10:36:40Z",
+      "startDateTime":"2023-09-25T09:28:38Z",
+      "endDateTime":"2023-09-25T09:28:41Z",
+      "organizer":{
+        "user":{
+          "id":"821809f5-0000-0000-0000-3b5136c0e777",
+          "tenantId":"dc368399-474c-4d40-900c-6265431fd81f"
         }
       },
-      "callee":{
-        "@odata.type":"#microsoft.graph.callRecords.participantEndpoint",
-        "cpuName":"Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz",
-        "cpuCoresCount":8,
-        "cpuProcessorSpeedInMhz":2295,
-        "userAgent":{
-          "@odata.type":"#microsoft.graph.callRecords.clientUserAgent",
-          "headerValue":"UCCAPI/16.0.12527.20122 OC/16.0.12527.20194 (Skype for Business)",
-          "platform":"windows",
-          "productFamily":"skypeForBusiness"
-        },
+      "organizer_v2":{
```

Comment:
This field is new, and in docs says that `organizer` and `participants` are [deprecated](https://learn.microsoft.com/en-us/graph/api/resources/callrecords-callrecord?view=graph-rest-1.0#properties)